### PR TITLE
Bump buildtools to 00097

### PR DESCRIPTION
**Pull request by ellismg**

### CODE REVIEW

**Comment by tarekgh** [comment]
LGTM. thanks for CC'ing me.

File: dir.props [hunk 1]
```diff
@@ -67,6 +67,7 @@
   <!-- list of nuget package sources passed to dnu -->
   <ItemGroup>
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
+
```
Comment (weshaggard) [inline dir.props]: nit: was there a reason for the extra line?

Comment (ellismg) [inline dir.props]: I had put another package source there for testing on Linux and when I removed it I missed deleting everything.

**Pull request merged**